.gitignore

```diff
@@ -1 +1,129 @@
-TODO
```

Comment:
Yeah exactly, unless there's anything specifically that needs ignoring in this repo, but not necessarily in all others.

(So for example, *.py[co] is not specific to this repo, those files should be ignored everywhere, in all repos. If you haven't seen it before by the way, my own global gitignore for example contains: https://github.com/Julian/dotfiles/blob/master/.config/git/ignore)